sensor_msgs/PointCloud2 base_cloud
sensor_msgs/PointCloud2 match_cloud